public/app/core/components/OptionsUI/slider.tsx

```diff
@@ -131,7 +131,7 @@ function getTextWidth(text: string, font: string): number | null {
 const getStylesSlider = (theme: GrafanaTheme2, width: number) => {
   return {
     numberInputWrapper: css`
-      margin-left: 10px;
+      margin-left: 24px;
```

Comment:
```suggestion
      margin-left: ${theme.spacing(3)};
```

Better to use the theme directly here for the spacing!

Reply:
Thanks for the suggestion, let me quickly check!